src/goose/utils/time_utils.py

```diff
@@ -0,0 +1,5 @@
+from datetime import datetime
+
+
+def formatted_time(time: datetime, format: str = "%Y-%m-%d %H:%M:%S") -> str:
```

Comment:
:nit: add `%Z` for timezones?

Reply:
just thought about it, we're basically using the iso format how about just using that instead? that way we can get rid of this util 

```python
datetime.now().astimezone().isoformat(timespec='seconds')
```

Reply:
> datetime.now().astimezone().isoformat(timespec='seconds')

I feel it is a bit overkill to add the timezone, etc since it is a desktop application at the moment. The logs are more for the end users.

Reply:
my point is the date format string can just use isoformat directly, why implement something that already exists? `astimezone()` is necessary to put it in local timezone.

i disagree that it's overkill it's part of a standard